tests/wpt/web-platform-tests/fetch/http-cache/heuristic.html

```diff
@@ -35,8 +36,7 @@
           {
             response_status: [299, "Whatever"],
             response_headers: [
-              ['Last-Modified', http_date(-3 * 100)],
-              ['Cache-Control', 'public']
+              ['Last-Modified', http_date(-3 * 100)]
```

Comment:
Why is this change required?

Reply:
The test reads "when Cache-Control: public is not present", and leaving the "public" header in the response makes the test always fail(since the response will be cached because of it).